community/SDK-GOVERNANCE.md

```diff
@@ -80,7 +80,7 @@ discussions". However, if the SDK is stable and does not need to be update then
 it might be determined that it current state is acceptable.
 
 To prevent the project from becoming _not actively maintained_, the community
-MAY takes the following actions:
+may takes the following actions:
```

Comment:
...the community may **take** the following...

Reply:
in next PR